tests/test_metadata.py

```diff
@@ -20,6 +20,20 @@ def metadata_model(helpers, data_model_labels):
         inputMModelLocation=helpers.get_data_path("example.model.jsonld"),
         data_model_labels=data_model_labels,
         inputMModelLocationType="local",
+        data_model_graph_pickle=helpers.get_data_path("example.model.pickle"),
+    )
+
+    return metadata_model
+
+
+def metadata_model_display(helpers, data_model_labels):
```

Comment:
why this is called `metadata_model_display`.. If you are using `data_model_labels`, and this could potentially be both "class_label" and "display_label", why is this named "display"?

Reply:
Could you add a comment to describe what this function is for.